milestone_1/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>The World Wide Web project</title>
+</head>
+<body>
+  <h1>World Wide Web</h1>
+  <p>
+    The WorldWideWeb (W3) is a wide-area <a href="http://info.cern.ch/hypertext/WWW/WhatIs.html"> hypermedia</a>
+    information retrieval initiative aiming to give universal access to a large universe of documents.
+  </p>
+  <p>
+    Everything there is online about W3 is linked directly or indirectly to this document, including an
+    <a href="http://info.cern.ch/hypertext/WWW/Summary.html">executive summary</a> of the project,
+    <a href="#">Mailing lists</a>, <a href="http://info.cern.ch/hypertext/WWW/Policy.html">Policy</a> ,
```

Comment:
@gorkhalitechie , why empty `#`?

Reply:
Sir, the href in the original web page doesn't has the link associated.